doc/stop.md

```diff
@@ -2,30 +2,59 @@
 @parent can-route.static
 @release 4.1
 
-Stops listening to the [can-route.data] observable and tears down any setup bindings.
+@description Stops listening to the [can-route.data] observable and tears down any setup bindings.
 
 @signature `route.stop()`
 
-Stops listening to changes in the URL as well as the observable defined in [can-route.data], and removes the current binding.
+  Stops listening to changes in the URL as well as the observable defined in [can-route.data], and removes the current binding.
 
-```js
-route.register( "{page}", { page: "home" } );
-route.start();
-route.data.page = "home";
-route.stop();
-route.data.page = "cart"; // hash is still #home
-```
+  ```html
+  <mock-url></mock-url>
+  <script type="module">
+  import "//unpkg.com/mock-url@^5.0.0";
+  import {route} from "can";
+
+  route.register("{page}", { page: "" });
+  route.start();
+  route.data.page = "home";
 
-@return {can-route} The can-route object.
+  // Changing the route is not synchronous
+  setTimeout(() => {
+    route.stop();
+    route.data.page = "cart"; // hash is still "home"
+    console.log( location.hash ) //-> "#!home"
+  }, 1000);
+
+  </script>
+  ```
+  @codepen
+
+  @return {can-route} The can-route object.
 
 @body
 
 ## Use
 
-If you need to disconnect an observable from the URL, call stop:
+If you need to disconnect an observable from the URL, call stop.
```

Comment:
Why would someone need to do that?  

The only use case I know is that you want to "turn off" routing because someone logged out.  For example, if someone logged out, you could call `route.stop()` and show a login modal.  No matter what url someone changed to, the page wouldn't actually change.  Then, when they have logged in, you can call `.start()`.  Maybe build something that shows this.

Reply:
Replaced example with one that matches closer to logging in/out